docs/tuning-apps/networking/ofi.rst

```diff
@@ -76,6 +78,11 @@ list. For example:
 
    shell$ ompi_info --param mtl ofi --level 9
 
+.. important:: When using the HPE CXI provider and ``mpirun`` as the job launcher,
```

Comment:
Just to be clear: this isn't entirely accurate. You could also simply add `--map-by :nouselocal` to the `mpirun` cmd line.

Reply:
yeah, if you want to get charged for N+1 vs N nodes where N is the number of nodes available for your MPI processes.

Reply:
I understand your point - however, your text doesn't indicate that you have a choice. You state that you *must* use the specified option - and that simply isn't true.

Reply:
That's a fair point.  It's probably worth pointing out the subtlety here -- i.e., what does this parameter mean and do, vs. what the user's options are if they don't use this MCA param.